Chef/Cookbooks/Relativity/recipes/post_relativity_powershell_module_setup.rb

```diff
@@ -23,7 +22,8 @@
   "Polly.dll", # Required for Import API
   "Relativity.Logging.dll", # Required for Import API
   "Relativity.Logging.Interfaces.dll", # Required for Import API
-  "Relativity.Kepler.dll"
+  "Relativity.Kepler.dll",

```

Comment:
Can we add a comment next to these two DLL's, which version we are using.

Reply:
Done